R/parttree.R

```diff
@@ -100,9 +103,16 @@ parttree =
                       keyby = .(node, var, side)]
 
     ## Get the coords data frame
+    if (flipaxes) {
+      var1 = 2
+      var2 = 1
+    } else {
+      var1 = 1
+      var2 = 2
+    }
     part_coords =
       part_dt[, `:=`(split = as.double(split))][
-        , `:=`(xvar = var == ..vars[1], yvar = var == ..vars[2])][
+        , `:=`(xvar = var == ..vars[var1], yvar = var == ..vars[var2])][
```

Comment:
Can you change lines 106-112 to:
```r
if (flipaxes)  vars = rev(vars)
```
That would also mean you can keep the existing line 115 as is (revert back to the original).